Chore/add credential request (#98)
UnumID/UnumID.github.io: docs/api-overview.mdx

The response body contains a list of all credentials on the `credentials` key.

---

### Get Partner Credential Requests

> > Get the partner pre-defined <Tip type="request">credential requests</Tip>.

Reviewer: ```suggestion
> Get the partner's pre-defined <Tip type="request">credential requests</Tip>.
```

**Method**: `GET` <br />
**Path**: `/credentialRequests/{partnerUuid}`

This endpoint returns a list of credential requests that the partner has defined for their brand.

#### Response Body:

```typescript title="CredentialRequest"
{
  "credentialRequests": {
    "type": string, // credential type
    "issuers": string[], // list of acceptable brandIds; if empty, all issuer brands are valid
    "required": boolean, // whether the credential is required
  }[]
}
```

:::note

> This endpoint is to partners use case. 
> More information about it can be found in the [Issuance Guide](/issuance-guide#already-issued-credentials)

Reviewer: ```suggestion
This endpoint is helpful when developing the partner issuance flow. More information about partner issuance can be found in the [Issuance Guide](/issuance-guide#partner-issuance).
```
